pyqtgraph/graphicsItems/ImageItem.py

```diff
@@ -262,8 +262,10 @@ def setRect(self, *args):
         This method cannot be used before an image is assigned.
         See the :ref:`examples <ImageItem_examples>` for how to manually set transformations.
         """
-        if isinstance(args, (QtCore.QRectF, QtCore.QRect)):
-            rect = args # use QRectF or QRect directly
+        if len(args) == 0:
```

Comment:
This should be "elif"

Reply:
Thank you!
The intention was indeed to reset the transform and be done. I have added a return statement for the len == 0 case. 

Sorry about the messed up tuple parsing. My mental stack overflowed on the logic to parse a tuple-with-a-tuple-in-it into a tuple-without-a-tuple-in-it. The messed up code passed tests locally, though, so I added some basic testing for the values / tuple / QRect / QRectF input cases.